view/adminhtml/templates/boltpay/button.phtml

```diff
@@ -3,9 +3,20 @@
  * @var \Bolt\Boltpay\Block\Form $block
  */
 $code = $block->escapeHtml($block->getMethodCode());
+$customerCreditCardInfos = $block->getCustomerCreditCardInfo();
 ?>
 
 <fieldset class="admin__fieldset payment-method" id="payment_form_<?php /* @noEscape */ echo $code; ?>" style="display:none">
+    <?php if($customerCreditCardInfos): ?>
+        <?php foreach ($customerCreditCardInfos as $customerCreditCardInfo): ?>
+        <div>
+            <select name="credit-cards" style="width: 284px; margin: 10px 0 5px;">
+                <option value=""><?php echo __('Add New Card') ?></option>
+                <option value="<?php echo $customerCreditCardInfo['id']  ?>"><?php echo 'XXXX'.'-'.@$customerCreditCardInfo['card_info']['last4'].' '.@$customerCreditCardInfo['card_info']['display_network'] ?></option>
```

Comment:
@johnmccombs1 can you add here `if` conditions instead of `@`?
It will be strange situation if we have inserted id, but other variables (`last4 `, `display_network `) would be empty.  
I prefer to show some message regarding missed values or errors.